Sources/SnowballSwiftKit/Views/Chip.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  Chip.swift
+//
+//
+//  Created by Vivian Phung on 6/30/23.
+//
+
+import SwiftUI
+import SnowballAssetKit
+
+public enum ChipSize {
+    case small
+    case medium
+}
+
+public struct Chip: View {
+    @Binding var isActive: Bool
+    var isDisabled: Bool
+    var title: String
+    var leftIcon: String?
+    var leftSystemIcon: String?
+    var rightIcon: String?
+    var rightSystemIcon: String?
+    var size: ChipSize
+
+    public init(isActive: Binding<Bool> = .constant(true),
+                isDisabled: Bool = false,
+                title: String,
+                leftIcon: String? = nil,
+                leftSystemIcon: String? = nil,
```

Comment:
what exactly does this mean. You can have two icons? Are they only ever images or image names or could they be views?